src/ui/components/SecondaryToolbox/redux-devtools/ReduxDevToolsContents.tsx

```diff
@@ -56,6 +63,34 @@ const PanelButton = ({ selected, children, name, onClick }: PanelButtonProps) =>
   );
 };
 
+function RDTInspector({
+  pauseId,
+  protocolValue,
+  path,
+}: {
+  path?: string;
+  pauseId: PauseId;
+  protocolValue: ProtocolValue | ProtocolNamedValue;
+}) {
+  const client = useContext(ReplayClientContext);
+  const clientValue = clientValueCache.read(client, pauseId, protocolValue);
+
+  const { objectId } = clientValue;
+
+  const objectWithPreview = objectCache.read(client, pauseId, objectId!, "canOverflow");
```

Comment:
:hand: One more caveat here. It _is_ possible, even though unlikely, that the top-level state value could be just a simple primitive (ie, `configureStore({reducer: counterReducer})`.  

In that case, there _won't_ be an `objectId` - it'll be a `value` instead.

We should handle that case as well.

Reply:
I think this case _should_ be handled now, but I am unable to test it out locally due to some Replay internal issue I seem to be facing.